setup.py

```diff
@@ -1,10 +1,27 @@
 #!/usr/bin/env python
 import os
+import sys
+
 from setuptools import setup, find_packages
 
+CURRENT_PYTHON = sys.version_info[:3]
+REQUIRED_PYTHON = (3, 6, 5)
+
+if CURRENT_PYTHON < REQUIRED_PYTHON:
+    sys.stderr.write(
+        f"Unsupported Python version* This version of Icon Service requires\n"
+        f"Python {'.'.join([str(v) for v in REQUIRED_PYTHON])}, "
+        f"but you're trying to install it on Python {'.'.join([str(v) for v in CURRENT_PYTHON])}. ")
+    sys.exit(1)
+
```

Comment:
I recommend to use `python_requires`.
https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires

Reply:
Thank you for providing better ways! I have applied tpython_requires.